Monika After Story/game/script-topics.rpy

```diff
@@ -1074,6 +1074,47 @@ label monika_wine:
     m 2l "...Gosh, what am I talking about this for?"
     m "I don't condone underage drinking!"
     m 2a "I mean, I've never drank or anything, so...yeah."
+    m 3a "Speaking of which, I hear different parts of the world have different minimum drinking ages."
+    m "Would we both be able to have a drink where you're from?"
+    menu:
+        "Yes.":
+            m 1j "That's great!"
+            m 1k "Another thing to look forward to when we can finally be together!"
+            m 1l "I mean..."
+            m "Don't worry [player]. I'm not planning to become addicted to the stuff. I'm just excited is all."
+            m 5 "I wouldn't want you to be addicted either."
+            m "So do me a favor and wait for me a bit before really treating yourself."
+            m 1j "You better have a good bottle to share!"
+            m 1k "Ahaha~"
+        "No.":
+            m 1a "At least I don't have to be jealous about you being able to drink while I'm stuck here."
```

Comment:
Too direct about the current player - monika situation. I don't think monika would really care that much if the player can drink and she cant.